e2e/testcafe-devextreme/tests/dataGrid/common/markup/T1286265_deletedRowHeight.ts

```diff
@@ -0,0 +1,83 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import DataGrid from 'devextreme-testcafe-models/dataGrid';
+import { createWidget } from '../../../../helpers/createWidget';
+import url from '../../../../helpers/getPageUrl';
+import { changeTheme } from '../../../../helpers/changeTheme';
+import { Themes } from '../../../../helpers/themes';
+
+fixture.disablePageReloads`DataGrid deleted row height consistency T1286265`
+  .page(url(__dirname, '../../../container.html'));
+
+const DATA_GRID_SELECTOR = '#container';
+// check first and second rows, because both can have issue independently each other
+const ROW_INDEXES = [1];
+
+[
+  Themes.fluentBlue,
+  Themes.fluentBlueCompact,
+  Themes.materialBlue,
+  Themes.materialBlueCompact,
+  Themes.genericLight,
+  Themes.genericLightCompact,
+].forEach((theme) => {
+  ROW_INDEXES.forEach((rowIndex) => {
+    test(`Row height should not change when marked as deleted - ${theme} - row - ${rowIndex}`, async (t) => {
```

Comment:
Also, let's remove the rowIndex from the test name -> it's redundant now

```suggestion
    test(`Row height should not change when marked as deleted - ${theme}`, async (t) => {
```